docs/src/pages/guides/FHIRPerformanceGuide.md

```diff
@@ -0,0 +1,800 @@
+CPU: user: 206.67 s, system: 113.27 s, elapsed: 537.14 s.
+INFO:  analyzing "fhirdata.logical_resources"
+INFO:  "logical_resources": scanned 30000 of 4210218 pages, containing 551774 live rows and 0 dead rows; 30000 rows in sample, 77436294 estimated total rows
+VACUUM
+```
+
+### 4.1.3 Transaction Id Wraparound
+
+Be ware of multixact wraparound issues, as highlighted by the following warning when running a manual vacuum:
+
+```fhirdb=> VACUUM (VERBOSE) fhirdata.logical_resources;
+WARNING:  oldest multixact is far in the past
+HINT:  Close open transactions with multixacts soon to avoid wraparound problems.
+```
+
+This indicates that the automatic vacuum process needs to be more aggressive. See [here](https://info.crunchydata.com/blog/managing-transaction-id-wraparound-in-postgresql) for details.
+
+
+### 4.1.4 Vacuum Monitoring
+
+Use the following query to see the impact of updates and deletes on the IBM FHIR Server tables (assuming the tenant is configured to use the `fhirdata` schema):
+
+```
+ SELECT relname, 
+        n_tup_ins AS "inserts", 
+        n_tup_upd AS "updates", 
+        n_tup_del AS "deletes", 
+        n_live_tup AS "live_tuples", 
+        n_dead_tup AS "dead_tuples"
+   FROM pg_stat_user_tables
+  WHERE schemaname = 'fhirdata'
+    AND (relname = 'logical_resources' OR relname LIKE '%_values')
+    AND n_dead_tup > 0;
+```
+
+The values reported are since the database was last restarted. To check uptime, run the following query:
+
+```
+ SELECT current_timestamp - pg_postmaster_start_time();
+    ?column?     
+-----------------
+ 05:45:41.835965
+```
+
+
+The following query can be used to see how many auto-vacuum jobs are currently in progress and for which tables:
+
+```
+fhirdb=> SELECT r.relname, v.* 
+           FROM pg_stat_progress_vacuum v, 
+                pg_stat_user_tables r 
+          WHERE r.relid = v.relid;
+
+          relname          |  pid   | datid | datname | relid |       phase       | heap_blks_total | heap_blks_scanned | heap_blks_vacuumed | index_vacuum_count | max_dead_tuples | num_dead_tuples 
+---------------------------+--------+-------+---------+-------+-------------------+-----------------+-------------------+--------------------+--------------------+-----------------+-----------------
+ imagingstudy_token_values | 451383 | 16478 | fhirdb  | 45603 | vacuuming indexes |          332918 |            332918 |                  0 |                  0 |        96879138 |            5102
+ condition_token_values    | 451793 | 16478 | fhirdb  | 52193 | scanning heap     |           53883 |             18173 |                  0 |                  0 |        15679953 |          301285
+ logical_resources         | 451180 | 16478 | fhirdb  | 16533 | vacuuming indexes |         1432256 |           1432256 |                  0 |                  0 |       178956970 |          765313
+```
+
+By default, only 3 vacuum jobs can run concurrently.
+
+
+
+## 4.2 IBM Db2
+
+TBD.
```

Comment:
maybe open a separate issue to fill this in eventually so that we don't forget...

Reply:
https://github.com/IBM/FHIR/issues/1890